css/style.css

```
@@ -0,0 +1,233 @@
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.main-wrapper {
+  width: 160rem;
+  margin: 0 auto;
+  position: relative
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.main-header:before {
+  content: '';
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: linear-gradient(to bottom, black -100%, transparent 90%);
+}
+
+.main-header {
+  width: 100%;
+  height: 12rem;
+  position: absolute;
+  display: flex;
+  padding-left: 18rem;
+  padding-right: 18rem;
+  gap: 3.8rem;
+  justify-content: space-between;
+}
+
+/******************************/
+/* LOGO AND NAVIGATION */
+/******************************/
+
+.logo {
+  font-family: "Yeseva One", sans-serif;
+  text-decoration: none;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 3.6rem;
+  opacity: 90%;
+  margin-right: auto;
+  align-self: center;
+  display: flex;
+}
+
+nav {
+  display: flex;
+  align-items: stretch;
+}
+
+.nav-bar {
+  list-style: none;
+  display: flex;
+  gap: 2.6rem;
+  align-items: stretch;
+}
+
+.nav-link,
+.nav-link:link,
+.nav-link:visited,
+.nav-link:active {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  padding-left: 1.2rem;
+  padding-right: 1.2rem;
+  text-decoration: none;
+  color: #fff;
+  font-style: normal;
+  position: relative;
+  font-size: 2.2rem;
+  font-weight: 600;
+  text-shadow: 1px 1px rgba(0, 0, 0, 0.3);
+}
+
+.nav-link.is-active {
+  cursor: default;
+}
+
+.nav-link:hover:before,
+.nav-link.is-active:before {
+  opacity: 70%;
+}
+
+.nav-link:before {
+  content: '';
+  width: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  border-top: 4px solid #fff;
+  transition: opacity 0.5s;
+  opacity: 0;
+}
+
+/******************************/
+/* SOCIAL-LINKS */
+/******************************/
+
+.social-links {
+  list-style: none;
+  display: flex;
+  gap: 1.2rem;
+  align-items: center;
+}
+
+.social-link {
+  display: flex;
+  width: 2rem;
+  height: 2rem;
+}
+
+.social-link:link,
+.social-link:visited {
+  color: #dab33e;
+}
+
+.social-link:hover {
+  opacity: 70%;
+  transition: opacity 0.3s;
+}
+
+.social-icon {
+  width: 2rem;
+  height: 2rem;
+}
+
+/******************************/
+/* PROMO SECTION */
+/******************************/
+
+.promo-section {
+  background: url('../gallery/banners/promo-banner.jpg');
+  background-size: cover;
+  padding-top: 12rem;
+  height: 75rem;
+}
+
+.promo-slogan-and-elements {
+  margin-top: 9.6rem;
+}
+
+.promo-slogan {
+  font-family: "Yeseva One", sans-serif;
+  text-transform: lowercase;
+  text-align: center;
+  font-size: 9rem;
+  line-height: 11.4rem;
+  font-weight: 100;
+  color: #f5f5f5;
+  text-shadow: 0.1rem 0.1rem rgba(0, 0, 0, 0.5);
+}
+
+.promo-decoration {
+  text-align: center;
+  margin-top: 2.8rem;
+}
+
+.decoration {
+  width: 23.4rem;
+}
```

Looks like we can remove container and leave only `img` with this class
```suggestion
.promo-decoration {
  margin-top: 2.8rem;
  width: 23.4rem;
}
```